src/components/PhotoAnalysis/PhotoAnalysis.tsx

```diff
@@ -50,6 +53,31 @@ const Analysis: React.FC = () => {
       .catch(err => setRekognitionResponse(JSON.stringify(err, null, 2)))
   }
 
+  const displayUploadButton = () => {
```

Comment:
@christoff-linde @brian-david @Jaceyjay 
I helped Nicole with this part, so I just wanted another opinion here. Any strong objections to hiding/showing the upload button in this way? I'm sure there are multiple ways to achieve this functionality; this just seemed to be the simplest

Reply:
IMHO it is a bit clunky to read, but that might just be me?

Reply:
Eh, we can just merge it 😃 too much brain power required to rethink